src/client/components/app/Transactions/Labs/LabDepositTx.tsx

```diff
@@ -107,14 +120,17 @@ export const LabDepositTx: FC<LabDepositTxProps> = ({ onClose }) => {
       return;
     }
 
-    dispatch(
-      VaultsActions.getExpectedTransactionOutcome({
-        transactionType: 'DEPOSIT',
-        sourceTokenAddress: selectedSellTokenAddress,
-        sourceTokenAmount: toWei(debouncedAmount, selectedSellToken.decimals),
-        targetTokenAddress: selectedLab.address,
-      })
-    );
+    if (simulationsEnabled) {
```

Comment:
```suggestion
    if (!simulationsEnabled) {
        return;
    }
```

Reply:
same as above about order of the next statement